internal/pkg/template/permissions_boundary_integration_test.go

```diff
@@ -0,0 +1,37 @@
+//go:build integration || localintegration
+
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package template
+
+import (
+	"bytes"
+	"github.com/stretchr/testify/require"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+)
+
+func TestPermissions_Boundary(t *testing.T) {
+	t.Run("every CloudFormation template must contain conditional permissions boundary field for all IAM roles", func(t *testing.T) {
+		cmd := exec.Command("find", "templates", "-name", "*yml")
+		output, err := cmd.Output()
```

Comment:
Is there a reason why we preffered using `find`? would this work on different platforms like Windows 🤔 
Can we leverage `os.ReadDir` and walk through the files that way?

Reply:
I switched it to `filepath.WalkDir`...what do you think?